examples/dub/simple_embedded/dub.json

```diff
@@ -6,7 +6,7 @@
         "targetType": "executable",
         "dflags-posix-dmd": ["-defaultlib=libphobos2.so"],
 	"dependencies": {
-            "pyd": "0.9.3",
```

Comment:
The proper way to do this would be to use a path-based dependency. E.g. this is how vibe-d does it: https://github.com/vibe-d/vibe.d/blob/a94f78ae55772933ec4d5c4aa411b4ec20033a46/examples/json/dub.json#L4-L7